postgresql/endpoint_methods.go

```diff
@@ -0,0 +1,56 @@
+package postgresql
+
+import (
+	"github.com/lib/pq"
+
+	"github.com/benchttp/server/benchttp"
+)
+
+func (c *computedStatsService) ListMetadataByUserID(userID int) ([]*benchttp.Metadata, error) {
```

Comment:
[suggestion] Let's avoid unnecessary pointers
```suggestion
func (c *computedStatsService) ListMetadataByUserID(userID int) ([]benchttp.Metadata, error) {
```